### PR TITLE
Adds listCampaigns to resto_json_api.md

### PR DESCRIPTION
**Pull request by tt-ghb**

### CODE REVIEW

File: specs/resto_json_api.md [hunk 1]
```diff
@@ -1332,6 +1368,114 @@ sample response:
   }
 }
 ```
+<a name="listcampaigns"></a>
```
Comment (m2ger) [inline specs/resto_json_api.md:1371]: anchor name should be `listCampaigns`

File: specs/resto_json_api.md [hunk 2]
```diff
@@ -307,6 +308,41 @@ See also [listCards](#listcards).
 * ``vatAmount`` - VAT amount in cents
 * ``vatCode`` - VAT code
 
+<a name="campaign"></a>
+### Campaign
+See also [listCampaigns](#listcampaigns).
+
+* ``campaignName`` - campaign Name
```
Comment (m2ger) [inline specs/resto_json_api.md:315]: campaign name

File: specs/resto_json_api.md [hunk 3]
```diff
@@ -34,7 +34,8 @@
     + [importCustomers](#importcustomers)
     + [listCards](#listcards)
     + [importCards](#importcards)
-
+    + [listCampaigns](#listCampaigns)
```
Comment (m2ger) [inline specs/resto_json_api.md:37]: should also list the new objects `Campaign` etc here

File: specs/resto_json_api.md [hunk 4]
```diff
@@ -307,6 +308,41 @@ See also [listCards](#listcards).
...
+
+* ``unitName`` - unit name, operational unit name in Restolution
+* ``restaurantID`` - restaurant ID, business unit code in Restolution
+* ``businessUnitUUID`` - globally unique identifier for this restaurant
+
+<a name="campaign-article-row"></a>
+### Campaign Article Row
+
+* ``articleID`` - sale article number
+* ``name`` - sale article name
```
Comment (m2ger) [inline specs/resto_json_api.md:338]: wondering whether `articleName` is more consistent; see Receipt Row

File: specs/resto_json_api.md [hunk 5]
```diff
@@ -1332,6 +1368,114 @@ sample response:
...
+					"operationalUnits": [
+						{
+							"unitName": "SC TEST",
+							"restaurantID": "1",
+							"businessUnitUUID": "dc06fe0e-b68b-4f66-a48e-2adc0ce56ffa"
+						}
+					]
+				}
+			],
+			"campaignsActiveForDate": "2020-10-05T00:00:00+03:00"
```
Comment (m2ger) [inline specs/resto_json_api.md:1474]: not described what exactly does this mean

**Review by m2ger — CHANGES_REQUESTED** [review]
listed some problems found in the initial review

**Review by hyvaril — APPROVED** [review]
Changes suggested in previous commit have been made.  Documentation seems adequate to me.

**Pull request merged**